app/src/main/res/values-uk/strings.xml

```diff
@@ -0,0 +1,162 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- This file contains resource definitions for displayed strings, allowing
+     them to be changed based on the locale and options. -->
+<resources xmlns:tools="http://schemas.android.com/tools" tools:ignore="MissingTranslation">
+    <!-- Simple strings. -->
+    <string name="mpv_activity">mpv</string>
+    <string name="title_activity_settings">Налаштування</string>
+    <string name="dialog_ok">OK</string>
+    <string name="dialog_cancel">Скасувати</string>
+    <string name="dialog_save">Зберегти</string>
+    <string name="dialog_yes">Так</string>
+    <string name="dialog_no">Ні</string>
+    <string name="dialog_prev">Попереднє</string>
+    <string name="dialog_next">Наступне</string>
```

Comment:
![grafik](https://user-images.githubusercontent.com/1042418/124505240-53430c00-ddc9-11eb-8b05-04603bf1a8f6.png)
These might be too long for the button, is there an abbreviation that can be used here?

Reply:
Yes